### DISCUSSION TITLE
Email Subscription

### DISCUSSION THREAD

**Discussion by HarryWangATX** [auroral-ui/hexo-theme-aurora#68] [Ideas]
I do not know if this is doable, but it would be really nice to have a subscription component that allows users to subscribe to your blog and get notified every time you post a new article.

**Comment by bennyxguo** [comment]
> I do not know if this is doable, but it would be really nice to have a subscription component that allows users to subscribe to your blog and get notified every time you post a new article.

This is not doable without the use of a backend, but I will drop this into the ideas, maybe I could figure a way to do it.

**Reply by HarryWangATX** [reply]
sounds good
